sleap/nn/system.py

```diff
@@ -195,6 +195,7 @@ def get_gpu_memory() -> List[int]:
         A list of the available memory on each GPU in MiB.
 
     """
+    os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
```

Comment:
Are you sure this environment variable shouldn't be set in the `get_all_gpus` function where we use tensorflow to get gpus? I think nvidia-smi already orders GPUs by PCI_BUS_ID.
https://stackoverflow.com/a/52816306